ynr/apps/uk_results/management/commands/import_2010_parl_results.py

```diff
@@ -0,0 +1,85 @@
+from django.core.management.base import BaseCommand
+from django.contrib.postgres.search import SearchQuery, SearchVector
+from slugify import slugify
+from candidates.models.popolo_extra import Ballot
+from django.db.models.functions import Length
+from popolo.models import Membership
+
+from uk_results.helpers import read_csv_from_url
+from uk_results import models
+
+
+class Command(BaseCommand):
+
+    EDGECASES = {
```

Comment:
This is fine for now, but a good example of why we should abstract the various list we have like this in to a set of matchers that we can reuse. For example, we already have matchers that deal with these cases, but they're not easy to use outside the SOPN importing process.